ReleaseProcedure.txt

```diff
@@ -25,9 +25,9 @@ Tools\gitreleasemanager\0.7.0\tools\GitReleaseManager.exe create -o nhibernate -
     If the release tag does not match the released version with major.minor.release
     formalism, the NuGet package will have an invalid link to release notes.
 
-  * Check/update version number in common.xml and NHibernate.props under
-    build-common folder, in master.xml under doc\reference folder, and in
-    appveyor.yml in the NHibernate root.
+  * Check/update version number in NHibernate.props under build-common folder.
+    Clear VersionSuffix if present.
+    NOTE: This paragraph is changed for 5.3 - if you are releasing earlier version please check instruction from appropriate branch   
```

Comment:
I do not think we really need that note, since normally, one would read this file from the branch on which he is issuing a release PR.

Reply:
Removed